src/tests/global_variables/env_variables_defaults.py

```diff
@@ -96,6 +96,8 @@ class _EnvVariables(DataPool, ABC):
         ["HOLD_INSTALLATION"], loader=lambda x: bool(strtobool(x)), default=env_defaults.DEFAULT_HOLD_INSTALLATION
     )
 
+    build_id: EnvVar = EnvVar(["BUILD_ID"])
```

Comment:
```suggestion
    ci_build_id: EnvVar = EnvVar(["BUILD_ID"])
```

Reply:
It could be any build-id not only ci. it's the id of the enclosing build 
anyway it's not critical to me. are you sure?